tests/test_session_relaunch_resumes_agent_sessions.py

```
@@ -0,0 +1,292 @@
+def main() -> int:
+    app_path_str = os.environ.get("CMUX_APP_PATH", "").strip()
+    if not app_path_str:
+        print("SKIP: set CMUX_APP_PATH to a built cmux DEV .app path")
+        return 0
+    app_path = Path(app_path_str)
+    if not app_path.exists():
+        print(f"SKIP: CMUX_APP_PATH does not exist: {app_path}")
+        return 0
+
+    bundle_id = _bundle_id(app_path)
+    socket_path = Path(f"/tmp/cmux-session-relaunch-agents-{bundle_id.replace('.', '-')}.sock")
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-session-relaunch-agents-"

(S108)